packages/ses/NEWS.md

```diff
@@ -2,9 +2,23 @@ User-visible changes in `ses`
 
 # Next release
 
-- The `evalTaming:` option `safe-eval` now can only throw error `SES_DIRECT_EVAL`.
+- The [evalTaming](https://github.com/endojs/endo/blob/master/packages/ses/docs/lockdown.md#evaltaming-options)
+  option `safe-eval` now can only throw error `SES_DIRECT_EVAL`.
   This allows SES to initialize with `'unsafe-eval'` or `no-eval` on hosts with no
```

Comment:
Still missing single quotes around no-eval.

```suggestion
  This allows SES to initialize with `'unsafe-eval'` or `'no-eval'` on hosts with no
```

Reply:
😅 updated and potentially autofixable with a markdownlint custom rule, noted in
- https://github.com/endojs/endo/issues/2759